### PR TITLE
refactor: show csv upload time

### PR DESCRIPTION
**Pull request by miles-grant-ibigroup**
### Checklist

- [ ] Appropriate branch selected _(all PRs must first be merged to `dev` before they can be merged to `master`)_
- [ ] Any modified or new methods or classes have helpful JSDoc and code is thoroughly commented
- [ ] The description lists all applicable issues this PR seeks to resolve
- [ ] The description lists any configuration setting(s) that differ from the default settings
- [ ] All tests and CI builds passing
- [ ] The description lists all relevant PRs included in this release _(remove this if not merging to master)_
- [ ] e2e tests are all passing _(remove this if not merging to master)_

### Description

This PR adds last modified dates to the csvs uploaded in the Pelias panel.

The dates are grabbed from the S3 headers. Not sure how we feel about that.

### CODE REVIEW

File: lib/manager/components/deployment/PeliasPanel.js [hunk 1]
```diff
@@ -29,13 +32,56 @@ type Props = {
...
+  }
+
+  // Only update if the deployment has changed
+  if (
+    this.props.deployment.peliasCsvFiles &&
+    this.props.deployment.peliasCsvFiles.every(
+      // $FlowFixMe flow is wrong
+      (file) => deployment.peliasCsvFiles.indexOf(file) > -1
+    ) &&
+    deployment.peliasCsvFiles &&
```
Comment (philip-cline) [inline lib/manager/components/deployment/PeliasPanel.js]: We've already checked `deployment.peliasCsvFiles` above, is that condition necessary here?

File: lib/manager/components/deployment/PeliasPanel.js [hunk 2]
```diff
@@ -175,6 +222,7 @@ class PeliasPanel extends Component<Props, State> {
         >
           Download
         </Button>
+        {label && <em style={{marginLeft: '1ch'}}>{label}</em>}
```
Comment (philip-cline) [inline lib/manager/components/deployment/PeliasPanel.js]: The text seems to need a `vertical-align: middle` in Firefox, but that's nothing serious

**Review by philip-cline — APPROVED** [review]
Looks good to me, couple tiny comments

File: lib/manager/components/deployment/PeliasPanel.js (hunk 1 above)
Comment (miles-grant-ibigroup) [reply lib/manager/components/deployment/PeliasPanel.js]: Good catch thanks! This was a leftover from earlier development

File: lib/manager/components/deployment/PeliasPanel.js (hunk 2 above)
Comment (miles-grant-ibigroup) [reply lib/manager/components/deployment/PeliasPanel.js]: Good catch thanks!

**Review by amy-corson-ibigroup — APPROVED** [review]

**Pull request merged**